<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Музей портативной электроники</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="timeline">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>50</y>
      <width>550</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1965</number>
    </property>
    <property name="maximum">
     <number>2010</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="timeSpinbox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1965</number>
    </property>
    <property name="maximum">
     <number>2010</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>35</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>1965</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>35</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>2010</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_notebook">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Ноутбуки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_pda">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>КПК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_smartphone">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Смартфоны</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_hpc">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Handheld PC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_calc">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Программируемые калькуляторы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_umpc">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>UMPC</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>timeline</sender>
   <signal>valueChanged(int)</signal>
   <receiver>timeSpinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeSpinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>timeline</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
